decidim-admin/config/routes.rb

```diff
@@ -18,18 +16,21 @@
       resources :user_roles, controller: "participatory_process_user_roles", only: [:destroy, :create, :index]
       resources :attachments, controller: "participatory_process_attachments"
       resources :features
-      resources :components, only: [:new, :create, :destroy]
     end
 
-    scope "/participatory_processes/:participatory_process_id/features/:feature_id/components/:current_component_id" do
-      Decidim.component_manifests.each do |manifest|
+    scope "/participatory_processes/:participatory_process_id/features/:feature_id/manage" do
+      Decidim.feature_manifests.each do |manifest|
         next unless manifest.admin_engine
-        constraints Decidim::Components::RouteConstraint.new(manifest) do
+
+        constraints lambda { |request|
+          feature = Decidim::CurrentFeature.new(request).call
```

Comment:
Can we keep the class for the constraint? The code reads really weird this way (lambda {} do)